### ISSUE TITLE
clarification on wg public and private keys

### ISSUE THREAD

**Issue by AndySchroder** [LightRider5/lnvpn#23]
On https://lnvpn.net/ you say that "Your private keys are only generated within the browser!". It's unclear to me about public keys. Do you generate a public key from my private key and then send that to your server in the background? Also, where is the public key for your end coming from? Is that constant, or fetched in the background from your server?

Also, https://lnvpn.net/api/documentation/ doesn't mention anything about the my public key. How are you getting my public key when the API is used?

**Comment by LightRider5** [comment]
Here is the file which is responsible for the key generation:
[wireguard.js](https://github.com/LightRider5/lnvpn/blob/main/client/src/wireguard.js)

As you see the public key is generated from the private key. This public key is send to our server as you said. 

The key generation for the API is different. Here it is a kind of custodial solution. You can't provide your own public key. But this will be possible in the future.

**Comment by AndySchroder** [comment]
Yes, please add it to the API query as an option to submit the public key. Then, the same query could be used to renew an existing connection (https://github.com/LightRider5/lnvpn/issues/22).
